app.py

```diff
@@ -3,26 +3,27 @@
 
 pygame.init()
 
-SCREEN_WIDTH, SCREEN_HEIGHT = 750, 750
+SCREEN_WIDTH, SCREEN_HEIGHT = 1200, 1000
```

Comment:
Have we considered the implications of doing this?

Reply:
I don't remember asking